src/ui/main_window.ui

```diff
@@ -359,6 +360,14 @@
     <string>Alt+V</string>
    </property>
   </action>
+  <action name="dactionInputLine">
```

Comment:
I can't remember but does this actually manifest as a visible widget in the main_window form/dialog?

Reply:
Yeah...